src/Microsoft.PowerShell.GraphicalTools/TypeGetter.cs

```diff
@@ -25,8 +25,17 @@ public FormatViewDefinition GetFormatViewDefinitionForObject(PSObject obj)
 
             var types = _cmdlet.InvokeCommand.InvokeScript("Get-FormatData " + typeName).ToList();
 
-            //No custom type definitions found
-            if (types == null || types.Count == 0) return null;
+            //No custom type definitions found - try the PowerShell specific format data
+            if (types == null || types.Count == 0) 
+            {
+                types = _cmdlet.InvokeCommand
+                    .InvokeScript("Get-FormatData -PowerShellVersion $PSVersionTable.PSVersion " + typeName).ToList();
```

Comment:
Because I've seen it called out on PSES PRs before, I wonder if this needs to be fully qualified (as `Microsoft.PowerShell.Utility\Get-FormatData`)

Reply:
updated!